### PR TITLE
Update package.json & package-lock.json

### PR DESCRIPTION
**Pull request by elvixpix**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,8 +4,9 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "export NODE_ENV=test && mocha api/tests/**/**.test.js",
-    "start": "export NODE_ENV=dev && nodemon server.js"
+    "test": "set NODE_ENV=test && mocha api/tests/**/**.test.js",
+    "start": "set NODE_ENV=dev&& nodemon server.js"
```
Comment (bionanek) [inline package.json]: adsdfsdfsfd

File: package.json [hunk 2]
```diff
@@ -29,10 +30,11 @@
   "dependencies": {
     "bcrypt": "^3.0.2",
     "body-parser": "^1.18.3",
-    "config": "^2.0.1",
+    "config": "^2.0.2",
```
Comment (bionanek) [inline package.json]: ??

File: package.json (hunk 1 above)
Comment (elvixpix) [reply package.json]: ddsaw

File: package.json (hunk 2 above)
Comment (elvixpix) [reply package.json]: !!

**Pull request merged**